apricot/local/service.go

```diff
@@ -477,6 +477,21 @@ func (s *Service) SetRuntimeEntry(component string, key string, value string) er
 	}
 }
 
+func (s *Service) GetRuntimeEntries(component string) ([]string, error) {
```

Comment:
I advise starting with `s.logMethod()` like the other handlers, just so the log output behaviour is consistent.